release-lockfile.spec.md

```diff
@@ -136,7 +136,7 @@ contract instances have an address on some specific chain.
 A name which refers to a specific *contract instance* on a specific chain from
 the deployments of a single release lockfile.  This name **must** be unique
 across all other *contract instances* for the given chain.  The name must
-conform to the regular expression `[a-zA-Z][a-zA-Z0-9_]*`.
+conform to the regular expression `[a-zA-Z][a-zA-Z0-9_]{0,255}`
```

Comment:
I chose 256 as an arbitrary upper limit on the allowed length.  Seems reasonable.